.teamcity/settings.kts

```diff
@@ -801,9 +801,10 @@ object CheckCodeStyleBranch : BuildType({
 	}
 })
 
-object RunCanaryE2eTests : BuildType({
-	name = "Canary e2e tests"
-	description = "Run canary e2e tests"
+object RunCalypsoE2eDesktopTests : BuildType({
```

Comment:
Object renamed because it is not about "canaries" anymore. The `uuid` parameter ensures TeamCity can understand this is a rename and not a brand new build.